addons/portal/static/src/js/portal_chatter.js

```diff
@@ -20,7 +21,8 @@ var PortalChatter = publicWidget.Widget.extend({
     template: 'portal.Chatter',
     xmlDependencies: ['/portal/static/src/xml/portal_chatter.xml'],
     events: {
-        "click .o_portal_chatter_pager_btn": '_onClickPager',
+        'click .o_portal_chatter_pager_btn': '_onClickPager',
+        'click .o_portal_chatter_js_is_internal': 'async _onClickUpdateIsInternal',
```

Comment:
I think since the event is on a div instead of the button, it will not correctly apply the async style (greying it if the rpc is too long) but oh well...